dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -4930,6 +4927,26 @@ paths:
       summary: Add an entry to a collection.
       tags:
         - organizations
+  /organizations/{organizationId}/deleteNonApprovedOrg:
+    delete:
+      description: Delete rejected organization
+      operationId: deleteRejectedOrPendingOrganization
+      parameters:
+        - description: Organization ID.
+          in: path
+          name: organizationId
+          required: true
+          schema:
+            format: int64
+            type: integer
+      responses:
+        default:
+          content:
+            application/json: {}
```

Comment:
Doesn't seem quite correct. I'd expect something like https://swagger.io/docs/specification/describing-responses/  Empty Response Body

Reply:
Still trying to figure it out... I suspected that the issue was the endpoint having a response of void.class, but that doesn't seem to be the case as the response body is still returning an empty object instead if it being just empty.

Reply:
You're looking for the openapi equivalent of:
https://github.com/swagger-api/swagger-core/issues/2446

Reply:
Issue has been addressed.